addons/storyshots/storyshots-puppeteer/src/axeTest.ts

```diff
@@ -11,13 +11,17 @@ declare global {
   }
 }
 
-export const axeTest = (customConfig: Partial<CommonConfig> = {}) =>
+export const axeTest = (customConfig: Partial<AxeConfig> = {}) => {
+  const config = { ...defaultAxeConfig, ...customConfig };
+  const { beforeAxeTest } = config;
+
   puppeteerTest({
```

Comment:
Did it actually work for you @coreyjv? I'm asking because the result of `puppeteerTest` is no longer returned, so instead of AXE it runs the default snapshot tests.

Ideally, we would catch that on CI, but unfortunately the corresponding GitHub Action was eventually removed: 
https://github.com/storybookjs/storybook/commit/9be0706518861b851b645b84a6bd0a52f7e78363

@ndelangen do you remember what exactly wasn't functioning back then?

Reply:
I'm sorry about that mistake @Hypnosphi! I didn't realize that this wouldn't be caught by CI. I can open another PR to address this if you'd like.

Reply:
No problem, I'm working on another PR for the same package, so I'll just include the fix there

Reply:
@Hypnosphi -- PR created to address the regression here: https://github.com/storybookjs/storybook/pull/15005

Again my apologies for not catching this the first time.